code/src/java/pcgen/persistence/GameModeFileLoader.java

```diff
@@ -408,7 +409,7 @@ private static GameMode loadGameModeMiscInfo(String aName, URI uri)
 		gameMode.setFeatTemplate(feat);
 
 		int[] dieSizes = gameMode.getDieSizes();
-		if (dieSizes == null || dieSizes.length == 0)
+		if (ArrayUtils.nullToEmpty(dieSizes).length == 0)
```

Comment:
Again, I think these need discussion.  This seems like churn and opinion, not changes that would be universally valuable.